cirq-core/cirq/ops/raw_types.py

```diff
@@ -596,6 +599,8 @@ def _commutes_(
 
         return np.allclose(m12, m21, atol=atol)
 
+    def equal_gates(self, other):
```

Comment:
Why is this here?  It looks like it could cause problems.

Reply:
Removed.